assets/js/dokan.js

```diff
@@ -1502,6 +1502,7 @@ jQuery(function($) {
         // Ajax search products tags
         $("#product_tag_search").select2({
             allowClear: false,
+            tags: true,
```

Comment:
No need any changes on this file

Reply:
Please change this code on file `product-editor.js` not `dokan.js`